docker/jmeter-master/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM platformhellofresh/kangal-jmeter:base
+
+ARG JMETER_VERSION=5.0
+
+ENV SSL_DISABLED false
+ENV WORKER_SVC_NAME jmeter-worker
+ENV TESTS_DIR /tests
+ENV SLEEP 5
+ENV USE_WORKERS false
+
+ENV AWS_DEFAULT_REGION ""
+ENV AWS_ENDPOINT_URL ""
+ENV AWS_BUCKET_NAME ""
```

Comment:
Please leave some comments why do we need this envs

Reply:
Added info to the README